wolfcrypt/src/port/atmel/atmel.c

```diff
@@ -915,62 +915,72 @@ int atcatls_verify_signature_cb(WOLFSSL* ssl, const byte* sig, unsigned int sigS
 
 static int atcatls_set_certificates(WOLFSSL_CTX *ctx) 
 {
-    #ifndef ATCATLS_TNGTLS_SIGNER_CERT_SIZE
-        #define ATCATLS_TNGTLS_SIGNER_CERT_SIZE 0x208
-    #endif
-    #ifndef ATCATLS_TNGTLS_DEVICE_CERT_SIZE
-        #define ATCATLS_TNGTLS_DEVICE_CERT_SIZE 0x222
-    #endif
-    #ifndef ATCATLS_TNGTLS_CERT_BUFF_SIZE
-        #define ATCATLS_TNGTLS_CERT_BUFF_SIZE (ATCATLS_TNGTLS_SIGNER_CERT_SIZE +\
-                                               ATCATLS_TNGTLS_DEVICE_CERT_SIZE)
-    #endif
-
     int ret = 0;
     ATCA_STATUS status;
-    size_t signerCertSize = ATCATLS_TNGTLS_SIGNER_CERT_SIZE;
-    size_t deviceCertSize = ATCATLS_TNGTLS_DEVICE_CERT_SIZE;
-    uint8_t certBuffer[ATCATLS_TNGTLS_CERT_BUFF_SIZE];
+    size_t signerCertSize=0;
+    size_t deviceCertSize=0;
+    uint8_t *certBuffer;
 
-    /*Read signer cert*/
-    status = tng_atcacert_read_signer_cert(&certBuffer[ATCATLS_TNGTLS_DEVICE_CERT_SIZE],
-                                           &signerCertSize);
+    /*fetch signer cert size*/
```

Comment:
For all new comments please add spaces before and after the multi-line comment. Example `/* Fetch signer cert size */`